modules/consensus/accept_test.go

```diff
@@ -1349,7 +1351,7 @@ func (cst *consensusSetTester) testPaymentChannelBlocks() error {
 		// txn needs to be fully custom. To get a custom txn, manually create an
 		// address and then use the wallet to fund that address.
 		channelSize := types.NewCurrency64(10e3)
-		channelFundingSK, channelFundingPK, err := crypto.GenerateSignatureKeys()
```

Comment:
no need to change the library call.

in the crypto package, define a struct that counts as the 'generic skg':

```
var (
    genericSKG SKG = NewSKG()
)

func GenerateSignatureKeys() stuff {
    return gerenicSKG.Generate()
}
```